README.md

```diff
@@ -63,13 +63,13 @@ sphinx-build -D language=pt_BR ./ _build/html/pt_BR # for *português do Brasil*
 * Access the documentation:
 
 ```bash
-open _build/html/index.html
+python3 -m http.server --d _build/html
```

Comment:
Apologies. I suggested a typo in my previous comment. Even though it works I think using `-d` is more correct.

```suggestion
python3 -m http.server -d _build/html
```